cli/azd/cmd/root.go

```diff
@@ -4,14 +4,23 @@
 package cmd
 
 import (
+	"context"
 	"fmt"
 	"os"
 
+	"github.com/azure/azure-dev/cli/azd/cmd/actions"
+	// Importing for infrastructure provider plugin registrations
```

Comment:
Importing these here allows the special `init()` functions to run to power the dynamic registration??

Reply:
Yup, that's how the plguni registration works.